src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbData/breadcrumbDataException.tsx

```diff
@@ -30,11 +36,61 @@ const BreadcrumbDataException = ({breadcrumb}: Props) => {
         })}
       {breadcrumb?.message &&
         getBreadcrumbCustomRendererValue({
-          value: breadcrumb.message,
+          value: (
+            <FormatMessage
+              event={event}
+              orgId={orgId}
+              breadcrumb={breadcrumb}
+              message={breadcrumb.message}
+            />
+          ),
           meta: getMeta(breadcrumb, 'message'),
         })}
     </BreadcrumbDataSummary>
   );
 };
 
+function isEventId(maybeEventId: string): boolean {
+  // maybeEventId is an event id if it's a hex string of 32 characters long
+  return /^[a-fA-F0-9]{32}$/.test(maybeEventId);
+}
+
+const FormatMessage = withProjects(function FormatMessageInner({
```

Comment:
@priscilawebdev I'm going to move this later. Let me know if this approach is fine or if a new breadcrumb type is preferable.

Reply:
yes...it's completely fine 🙂

Reply:
Did you consider using `withOrganization()` instead of prop drilling the `orgId`?

Reply:
Yea I did. Do you feel that's better?

Reply:
You would have to drill the event anyways, so the orgId isn't a big change.